app/static/js/script.js

```diff
@@ -3,20 +3,18 @@
 	// Initialize appliciation
   	var app = {
 		init: function() {
-			window.onload = () => {
-				location.hash && sections.toggle(location.hash)
-			}
+			location.hash && sections.toggle(location.hash)
 			routes.init();
 			// Global app stuff
 		}
 	}
 	// Handle routes and states
 	var routes = {
 		init: function() {
-			onhashchange = () => {
```

Comment:
An event listener is usually preferred. Also the correct syntax would be `windoe.onhashchange`